Content.Shared/Cloning/CloningSettingsPrototype.cs

```diff
@@ -62,11 +64,20 @@ public sealed partial class CloningSettingsPrototype : IPrototype, IInheritingPr
 
     /// TODO: Make this not a string https://github.com/space-wizards/RobustToolbox/issues/5709
     /// <summary>
-    ///     Components to copy from the original to the clone.
-    ///     This only makes a shallow copy of datafields!
-    ///     If you need a deep copy or additional component initialization, then subscribe to CloningEvent instead!
+    ///     Components to copy from the original to the clone using CopyComp.
+    ///     This makes a deepcopy of all datafields, including information the clone might not own!
+    ///     If you need to exclude data or do additional component initialization, then subscribe to CloningEvent instead!
+    ///     Components in this list that the orginal does not have will be removed from the clone.
     /// </summary>
-    [DataField]
+    [DataField(customTypeSerializer: typeof(CustomHashSetSerializer<string, ComponentNameSerializer>))]
     [AlwaysPushInheritance]
     public HashSet<string> Components = new();
+
+    /// <summary>
+    ///  Components to remove from the clone and copy over manually using a CloneEvent raised on the original.
+    ///  Use this for example when the component cannot be copied using CopyComp, for example when having an Uid as a datafield.
```

Comment:
```suggestion
    ///  Use this when the component cannot be copied using CopyComp, for example when having an Uid as a datafield.
```
typo from me earlier